cmd/kwil-admin/nodecfg/toml.go

```diff
@@ -118,6 +118,16 @@ pg_db_pass = "{{ .AppCfg.DBPass }}"
 # PostgreSQL database name (override database name, default is "kwild")
 pg_db_name = "{{ .AppCfg.DBName }}"
 
+# The admin RPC server can require a password, if set. Ensure the connection is
+# encrypted since the password is sent unencrypted in the HTTP Authorization
+# header. Not needed if client authentication is done with mutual TLS (clients.pem).
+# admin_pass = "{{ .AppCfg.AdminRPCPass }}"
```

Comment:
Oh, I haven't tested this password based auth.  Was just an idea I wanted to try since it's simpler than TLS key pairs if a deployment happens to have a secured network or using TLS with a root CA (not self-signed) an terminated at the edge of a secure network.